### PR TITLE
Fixed docs.google.com

### PR DESCRIPTION
**Pull request by sprinkledcheese**
- Google forms logo.

Example link for icons fix: https://docs.google.com/viewer?a=v&pid=sites&srcid=ZWhvcm5ldHMub3JnfGVmbXMtbGFuZ3VhZ2UtYXJ0c3xneDo0YmRkZjY2MDAxMTIwY2Vh

### CODE REVIEW

**Comment by Gusted** [comment]
I was unable to find both CSS Selectors.

**Comment by sprinkledcheese** [comment]
Google forms logo isn't in that example link. 
Here: https://docs.google.com/forms/d/e/1FAIpQLSeI8_vYyaJgM7SJM4Y9AWfLq-tglWZh6yt7bEXEOJr_L-hV1A/viewform

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -893,6 +894,9 @@ CSS
 .docs-text-ui-cursor-blink {
     fill: ${black} !important;
 }
+.ndfHFb-c4YZDc.ndfHFb-c4YZDc-i5oIFb .ndfHFb-c4YZDc-Bz112c:not([onclick]):not(:link):not(:visited) {
+ background-image: url('//ssl.gstatic.com/docs/common/viewer/v3/v-sprite26.svg')!important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:898]: 4 Spaces
Space between `)` and `!important`

Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config]: Sorry for the trouble, didn't check this one.

**Pull request merged**